Kamrans changes

KamranAzeem opened the pull request:
Hi Sofus, Mike,
I am not sure why my changes were removed from master and put aside as kamran's changes. I would love to learn / fix - if I did something wrong! At the moment, I have no clue! :(

I made several improvements:
* Moved kubectl setup to a separate file.
* Improved the first exercise for pods, deployments, services, high availability and namespaces
* Introduced rolling updates
* Setup more easy to understand /practical secrets example
* Added an exercise to use storage (PV and PVC)
* Updated README to have a list of exercises as Table of Contents
* Kept all other exercises created by earlier team members, nothing deleted, e.g. ingresses. 

All of above are actually aligned to my presentation I delivered at Day of Cloud Native 2018.
I hope you will find this useful!

meekrosoft commented:
I am giving a kubernetes training today and your changes didn't work on the lab setup.  For instance the first step of setting up kubectl...

I didn't have time to go through all your changes (which were very difficult to diff) so I put your stuff on a branch and reverted master until we could find time to go through together.  Let's talk next week to see how we can do this in a controlled way

KamranAzeem commented:
Sure!
Actually, setting up kubectl is not what I wrote. I just moved it to a
different file. I may have adjusted few things there, but I am not sure.
But exercise 1 -4 in the presentation are there, and they work 100%

Regards,
Kamran

On Fri, Apr 20, 2018, 08:42 Mike Long <notifications@github.com> wrote:

> I am giving a kubernetes training today and your changes didn't work on
> the lab setup. For instance the first step of setting up kubectl...
>
> I didn't have time to go through all your changes (which were very
> difficult to diff) so I put your stuff on a branch and reverted master
> until we could find time to go through together. Let's talk next week to
> see how we can do this in a controlled way
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/praqma-training/kubernetes-katas/pull/4#issuecomment-382996246>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEBWMRoELbjFkej6ThJV1TyktROVKc0Hks5tqYM8gaJpZM4TckUv>
> .
>
